deploy/cr.yaml

```diff
@@ -81,38 +81,37 @@ spec:
   #   Optional: resource settings for OneAgent container. Consumption of the OneAgent heavily depends
   #   on the workload to monitor; please adjust values accordingly.
   #
-  #   resources:
-  #     requests:
-  #       cpu: 100m
-  #       memory: 512Mi
-  #     limits:
-  #       cpu: 300m
-  #       memory: 1.5Gi
+  #  resources:
+  #    requests:
+  #      cpu: 100m
+  #      memory: 512Mi
+  #    limits:
+  #      cpu: 300m
+  #      memory: 1.5Gi
 
   #   Optional: labels are customer defined labels for OneAgent Pods to structure workloads as desired.
   #
-  #   labels:
-  #     custom: label
+  #  labels:
+  #    custom: label
 
   #   Optional: arguments to add to the ActiveGate instances.
   #   Available options: https://www.dynatrace.com/support/help/shortlink/linux-custom-installation
   #   Limitations: https://www.dynatrace.com/support/help/shortlink/oneagent-docker#limitations
   #
-  #   args:
-  #     - "LOG=<folder>"
+  #  args: []
 
   #   Optional: additional environment variables to add to the ActiveGate Pods.
   #
   #   env: []
 
   #   Optional: Set activation group for ActiveGate
   #
-  #   group: ""
+  #  group: "mygroup"
```

Comment:
Please don't put anything here - there's a default value called "default" which gets automatically applied in case nothing is set.